coalib/bearlib/languages/definitions/ZShell.py

```diff
@@ -0,0 +1,19 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class ZShell:
+    extensions = ('.zsh',)
+    comment_delimiters = '#',
```

Comment:
on the line above, you use round brackets.  here you dont.  self-consistency is important.

check what style flake8 plugin and pypi package `hacking` uses for tuples?

there is also `flake8-comprehensions` which might have a preference.

(we dont use flake8, or its many useful plugins, but we should..)